modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/string.ts

```diff
@@ -12,6 +12,24 @@
  * details.
  */
 
+const defaultLanguageId = Liferay.ThemeDisplay.getDefaultLanguageId();
+
+type Label = {
+	[key: string]: string;
+};
+
+/**
+ * Get the label according to the locale
+ */
+
+export function getLocale(
+	labels: Label,
+	creationLanguageId: string,
```

Comment:
hey @alinedoleron, to further specify the type of `creationLanguageId`, we can use an existing type which is: `Locale`